src/tests/ftest/soak/utils.py

```diff
@@ -184,7 +184,9 @@ def get_remote_dir(self, source_dir, dest_dir, host_list, shared_dir=None,
             try:
                 run_command(command, timeout=30)
             except DaosTestError as error:
-                raise SoakTestError("<<FAILED: job logs failed to copy>>") from error
+                raise SoakTestError(
+                    "<<FAILED: Soak logfiles not copied from shared area>>: {}") from error
```

Comment:
Were you intending to include `shared_dir_tmp` in the error message?
```suggestion
                    "<<FAILED: Soak logfiles not copied from shared area>>: {}".format(shared_dir_tmp)) from error
```